lib/ansiblelint/cli.py

```diff
@@ -51,11 +52,10 @@ def load_config(config_file):
 
     if config_file:
         if not os.path.exists(config_path):
-            print(
-                "Config file not found '{cfg!s}'.".format(cfg=config_path),
-                file=sys.stderr,
+            logger.info(
+                "Config file not found '{cfg!s}'.".format(cfg=config_path)
```

Comment:
Linters (pylint?) usually yell at manual interpolation in logger calls, it's preferable to pass arguments separately. Would something like this work?
```suggestion
                "Config file not found '%(cfg)s'.", cfg=config_path)
```

Reply:
Clearly it would work. I did not touch it because use of new formatter was your suggestion last week. Another alternative is to use one of the hackish solutions that add formatting support to logging (a custom formatter). Still, i am bit afraid that we may endup introducing more problems than solving. Better to go to older but fully supported syntax.

Reply:
Oh, let me make it clear then. Formatting of the usual strings should be done with the use of `str.format()` (and f-strings in the future) but it's a bit different for logging: when you pass a separate pre-rendered string it doesn't know about the vars but when you pass them separately it does. I think the idea of separating the args comes from structlog.

Also, pylint has [logging-not-lazy (W1201) rule](http://pylint.pycqa.org/en/latest/technical_reference/features.html#logging-checker-messages) that implies that lazy interpolation is preferable (meaning that it won't happen if not really necessary per logging config).